src/GeneralStateTestsFiller/stCreateTest/CREATE_EOF1Invalid_FromEOFFiller.yml

```diff
@@ -91,6 +91,142 @@ CREATE_EOF1Invalid_FromEOF:
+      - '0xef000101002a007fef000101001b0060016001550000000000000000000000000000000000000000600052600d6000f37b'
+      # mstore(0x00, 0xef000101001b0060016001550000000000000000000000000000000000000000) return(0x00, 0x0d) push28(0x00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3
+      - '0xef0001010045007fef000101001b0060016001550000000000000000000000000000000000000000600052600d6000f37b00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3'
+      # mstore(0x00, 0xef000101001b0060016001550000000000000000000000000000000000000000) return(0x00, 0x0d) push29(
+      - '0xef000101002a007fef000101001b0060016001550000000000000000000000000000000000000000600052600d6000f37c'
+      # mstore(0x00, 0xef000101001b0060016001550000000000000000000000000000000000000000) return(0x00, 0x0d) push29(0x00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fd
+      - '0xef0001010046007fef000101001b0060016001550000000000000000000000000000000000000000600052600d6000f37c00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fd'
+      # mstore(0x00, 0xef000101001b0060016001550000000000000000000000000000000000000000) return(0x00, 0x0d) push30(
+      - '0xef000101002a007fef000101001b0060016001550000000000000000000000000000000000000000600052600d6000f37d'
+      # mstore(0x00, 0xef000101001b0060016001550000000000000000000000000000000000000000) return(0x00, 0x0d) push30(0x00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfe
+      - '0xef0001010047007fef000101001b0060016001550000000000000000000000000000000000000000600052600d6000f37d00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfe'
+      # mstore(0x00, 0xef000101001b0060016001550000000000000000000000000000000000000000) return(0x00, 0x0d) push31(
+      - '0xef000101002a007fef000101001b0060016001550000000000000000000000000000000000000000600052600d6000f37e'
+      # mstore(0x00, 0xef000101001b0060016001550000000000000000000000000000000000000000) return(0x00, 0x0d) push31(0x00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff
+      - '0xef0001010048007fef000101001b0060016001550000000000000000000000000000000000000000600052600d6000f37e00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff'
+      # mstore(0x00, 0xef000101001b0060016001550000000000000000000000000000000000000000) return(0x00, 0x0d) push32(
+      - '0xef000101002a007fef000101001b0060016001550000000000000000000000000000000000000000600052600d6000f37f'
+      # mstore(0x00, 0xef000101001b0060016001550000000000000000000000000000000000000000) return(0x00, 0x0d) push32(0x00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00
+      - '0xef0001010049007fef000101001b0060016001550000000000000000000000000000000000000000600052600d6000f37f00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00'
+      # EIP-3670: Invalid EOF initcode - Containing undefined instruction (0xf6)
+      # { mstore(0x00, 0xef000101001b0060016001550000000000000000000000000000000000000000) return(0x00, 0x0d) 0xf6 }
```

Comment:
Add one with undefined opcode before terminating instruction (or just change this one)